tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "files": [],
+  "extends": "./tsconfig-base.json",
```

Comment:
I didn't think we wanted this here, since this `tsconfig.json` actually doesn't apply to any files (it just has references to the other file locations)

Reply:
removed.